commands/previewPush.go

```diff
@@ -130,19 +130,21 @@ func (args *PushArgs) flags() []cli.Flag {
 }
 
 // Preview implements the preview subcommand.
-func Preview(args PreviewArgs) error {
+func Preview(args PPreviewArgs) error {
+	fmt.Fprintf(os.Stderr, "DEBUG: OLD PREVIEW (delete before merge)\n")
```

Comment:
A GitHub comment so that `delete before merge` action is guaranteed/cannot be forgotten.